composer.json

```
@@ -37,7 +37,7 @@
     "require-dev": {
         "doctrine/coding-standard": "^6.0",
         "phpstan/phpstan": "^0.12.18",
-        "phpunit/phpunit": "^7.5",
+        "phpunit/phpunit": "^8.5|^9.5",
```

```suggestion
        "phpunit/phpunit": "^9.4",
```

There is no PHPUnit 9.5 yet. PHPUnit 9.4 should be compatible with php 7.3, so no need for PHPUnit 8 anymore.